gulpfile.coffee

```diff
@@ -165,6 +165,24 @@ taskConfigs =
       src: getGlob('public_conf/*.coffee')
       dest: build + '/src/javascripts/ServerAPI'
       options: _.extend _.clone(globalCoffeeOptions), {}
+    ,
+      taskName: "buildUtilities"
+      src: getGlob('modules/BuildUtilities/src/server/*.coffee')
+      dest: build + '/src/javascripts/BuildUtilities'
+      options: _.extend _.clone(globalCoffeeOptions), {}
+      renameFunction: getFirstFolderName
+    ,
+      taskName: "serverAPI"
+      src: getGlob('modules/ServerAPI/src/server/*.coffee')
+      dest: build + '/src/javascripts/ServerAPI'
+      options: _.extend _.clone(globalCoffeeOptions), {}
+      renameFunction: getFirstFolderName
+    ,
+      taskName: "serverUtilityFunctions"
+      src: getGlob('modules/**/src/server/routes/ServerUtilityFunctions.coffee')
+      dest: build + '/routes'
+      options: _.extend _.clone(globalCoffeeOptions), {}
+      flatten: true
```

Comment:
Changes to make it possible to run prepare config files right at the start of the instance which elminates the possibility of configs not being Prepared before ACAS starts